js/menu.js

```diff
@@ -0,0 +1,124 @@
+
+// create works
+function generateWorks({ name, tech, ImgUrl }, index) {
+  const workContainer = document.createElement('div');
+  workContainer.classList.add('works-item');
+
+  workContainer.innerHTML = `
+  <div class="work-img">
+    <div class="proj-image">
+      <img src="Images/${ImgUrl}" alt="${name}" />
+    </div>
+  </div>
+  <div class="lower">
+    <h3>${name}</h3>
+    <ul class="tags">
+    ${tech.map(
+    (tec) => `<li>
+      <a href="#"> ${tec} </a>
+    </li>`,
+  )}
+    </ul>
+    <button class="btn show-modal" data-id="${index}" type="button">
+      See Project
+    </button>
+  </div>
+  `;
+
+  workOuterContainer.appendChild(workContainer);
+}
+
+// create works section
+workData.forEach((work, index) => {
+  generateWorks(work, index);
+});
+
+// hambuger menu imports
+const btn = document.querySelector('.mb-menu');
+const menu = document.querySelector('.hamburger');
+const menuList = document.querySelectorAll('.mb-menu-list a');
+
+// hambuger menu implementation
+btn.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  document.body.classList.toggle('no-scroll');
+});
+
+menuList.forEach((list) => {
+  list.addEventListener('click', () => {
+    menu.classList.remove('active');
+  });
+});
+
+// modal implementation
+const modalContainer = document.querySelector('.modal-container');
+const workBtns = document.querySelectorAll('.show-modal');
+
+// create modal
+function generateModal({
+  ImgUrl, name, description, tech, demoUrl, gitUrl,
+}) {
+  const modal = document.createElement('div');
+  modal.classList.add('modal');
+
+  modal.innerHTML = `
+    <button type="button" class="modalcross">
+      <span></span>
+      <span></span>
+    </button>
+    <div class="modal-img">
+      <img src="Images/${ImgUrl}" alt="${name}" />
+    </div>
+    <div class="modal-d">
+      <h2 class="modal-title">${name}</h2>
+      <div class="modal-btn desktop">
+        <a href="${demoUrl}" class="btn"
+          ><span>See Live</span> <img src="Images/export.svg" alt="see live"
+        /></a>
+        <a href="${gitUrl}" class="btn"
+          ><span>See Source</span> <img src="Images/github_ic.svg" alt="see live"
+        /></a>
+      </div>
+    </div>
+    <ul class="tags">
+      ${tech.map(
+    (tec) => `<li>
+        <a href="#"> ${tec} </a>
+      </li>`,
+  )}
+    </ul>
+    <p class="modal-desc">
+      ${description}
+    </p>
+    <div class="modal-btn mobile">
+      <a href="${demoUrl}" class="btn"
+        ><span>See Live</span> <img src="Images/export.svg" alt="see live"
+      /></a>
+      <a href="${gitUrl}" class="btn"
+        ><span>See Source</span> <img src="Images/github_ic.svg" alt="see live"
+      /></a>
+    </div>
+  `;
+
+  modalContainer.appendChild(modal);
+}
+
+function closeBtn() {
+  document.querySelector('.modalcross').addEventListener('click', () => {
+    modalContainer.classList.remove('active');
+    document.body.classList.remove('no-scroll');
+  });
+}
+
```

Comment:
- Kindly try to match the Figma design for the left and right spacing for the pop-up modal.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/156469825-689675dc-6f9c-4bc4-9a47-3bc81b503678.png) | ![image](https://user-images.githubusercontent.com/24703171/156469888-c8b9b946-d74d-4814-bdb0-d66e3080f4b3.png) |

- Kindly try to match the close button in the mobile view to match the Figma design.
- Kindly add the ` border-radius ` for the pop-up modal to match the Figma design.
- Please try to display the image for the `see source` button.
![image](https://user-images.githubusercontent.com/24703171/156470244-28a0d0a6-e9dd-4d9d-9613-df9cb5481a01.png)
- Kindly try to remove the comma between the technology list in the pop-up modal.
![image](https://user-images.githubusercontent.com/24703171/156470648-cb055b54-cee7-4854-8157-663560f50964.png)

Reply:
Hi, thanks for the feedback, I fixed all the above points, but regarding the button view, I cannot use white or black background due to the fact that it clashes with the IMG or body color, please check my changes before I request for the last review.